c/meterpreter/source/extensions/powershell/powershell_bridge.cpp

```diff
@@ -538,6 +569,151 @@ DWORD powershell_channel_close(Channel* channel, Packet* request, LPVOID context
 	return ERROR_SUCCESS;
 }
 
+DWORD channelise_session(wchar_t* sessionId, Channel* channel, LPVOID context)

```

Comment:
you win this time channelise

Reply:
Cleary @OJ named this function.

Reply:
Clearly @OJ named this function.

Reply:
Don't abuse our beloved language with unnecessary zzzzzz's!